test/helpers/azuremonitor/mockServer.go

```diff
@@ -0,0 +1,154 @@
+package azuremonitor
+
+import (
+	"context"
+	"crypto/tls"
+	_ "embed"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/openshift/cluster-logging-operator/test"
+	"github.com/openshift/cluster-logging-operator/test/helpers/oc"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/cluster-logging-operator/internal/utils"
+
+	"github.com/openshift/cluster-logging-operator/internal/runtime"
+
+	openshiftv1 "github.com/openshift/api/route/v1"
+	"github.com/openshift/cluster-logging-operator/test/client"
+	"golang.org/x/sync/errgroup"
+	corev1 "k8s.io/api/core/v1"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//go:embed azure-http-data-collector-api.json
+var AzureHttpDataCollectorApi string
+
+const (
+	image         = "mockoon/cli:latest"
```

Comment:
lets tag a version of this and push it to quay.io/openshift-logging so we can ensure it will not change from under us

Reply:
done